### ISSUE TITLE
Unexpected one line copy/paste behavior only in Java source code in VS Code

### ISSUE THREAD

**Issue by yuma-fm** [redhat-developer/vscode-java#3445]
[provide a description of the issue]
With the `Language Support for Java(TM) by Red Hat` installed in VS Code, moving the cursor to any line of Java source code and copying the line with `Ctrl + C` and pasting it with `Ctrl + V` without selecting characters causes unintended behavior.

##### Environment
- Operating System: Windows 11 Pro 23H2 22631.2861
- JDK version: 17.0.3.7
- Visual Studio Code version: 1.85.1
- Java extension version: 1.25.1

##### Steps To Reproduce
1. Start VS Code
2. Install `Language Support for Java(TM) by Red Hat`
3. Restart VS Code
4. Open a file with any `.java` extension
5. Copy and paste one line

##### Current Result
![current](https://github.com/redhat-developer/vscode-java/assets/108521255/26dc7adf-5c50-4d74-8ca0-57737ffeaaba)

##### Expected Result
※ Behavior when uninstalling the `Language Support for Java(TM) by Red Hat`.
![expect](https://github.com/redhat-developer/vscode-java/assets/108521255/d5dfc46d-dedc-4ca1-8535-67febee850a2)

##### Additional Informations
This text is written in Japanese and translated at DeepL.

**Comment by rgrunber** [comment]
As a workaround, could you try adding the following to either your workspace or user settings :

```json
"[java]": {
        "editor.pasteAs.enabled": false
}
```

It should disable our paste support for the Java language only. If the problem no longer occurs, it's definitely related to vscode-java.

I'm not yet able to reproduce this so it could be a combination of either Windows 11 or the DeepL extension (assuming that's what you have ?) inserting additional characters. With that said, I think this might get solved by fixing https://github.com/redhat-developer/vscode-java/issues/3444 . Our paste support should do nothing in cases where there's no interesting content to adjust.

**Comment by yuma-fm** [comment]
thanks!!
By applying the above workaround, the problem is no longer reproduced. 🙇‍♂️

**Comment by fbricon** [comment]
While I can't reproduce the problem, the fix for #3444 surely also fixes this, as pasting no longer shows vscode-java involved, i.e. the pasting icon selector is no longer visible:
<img width="115" alt="Screenshot 2024-01-11 at 09 04 25" src="https://github.com/redhat-developer/vscode-java/assets/148698/bd57d906-24db-4513-8feb-e7cf28e1a75b">

**Comment by yuma-fm** [comment]
After switching to version `v1.26.2024011008` and removing the following options, this issue was not reproduced!
```json
"[java]": {
    "editor.pasteAs.enabled": false
}
```
Thank you🙇‍♂️
